rdbms/models/languages.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import Column, Integer, Text, ForeignKey
+from sqlalchemy.exc import IntegrityError
+
+from .database import Base
+from .database import session
+from .users import Users
+
+class Languages(Base):
+
+    __tablename__='languages'
+
+    id = Column(Integer, nullable=False, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    languages = Column(Text)
```

Comment:
Use String instead of Text for short sequences of variable characters.

Reply:
Should be language, not languages. There is one language name per record, and users can have multiple records.